optimum/intel/openvino/quantization.py

```diff
@@ -1172,31 +1135,80 @@ def _hybrid_quantization(
     Returns:
         The OpenVINO Runtime model with applied hybrid quantization.
     """
-    ops_to_compress = _collect_ops_with_weights(model)
-
-    wc_config = copy.deepcopy(quantization_config)
-    wc_config.ignored_scope = wc_config.ignored_scope or {}
-
-    wc_ignored_types = ["Convolution"] if any(op.get_type_name() == "Convolution" for op in model.get_ops()) else []
-    wc_config.ignored_scope["types"] = wc_config.ignored_scope.get("types", []) + wc_ignored_types
-    compressed_model = _weight_only_quantization(model, wc_config, **kwargs)
-
-    ptq_ignored_scope = quantization_config.get_ignored_scope_instance()
-    ptq_ignored_scope.names += ops_to_compress
-
-    subset_size = quantization_config.num_samples if quantization_config.num_samples else 200
-    quantized_model = nncf.quantize(
-        model=compressed_model,
-        calibration_dataset=dataset,
-        model_type=nncf.ModelType.TRANSFORMER,
-        ignored_scope=ptq_ignored_scope,
-        # SQ algo should be disabled for MatMul nodes because their weights are already compressed
-        advanced_parameters=nncf.AdvancedQuantizationParameters(
-            smooth_quant_alphas=AdvancedSmoothQuantParameters(matmul=-1)
-        ),
-        subset_size=subset_size,
+
+    wc_config = quantization_config.clone()
+    wc_config.ignored_scope = {}
+    if any(op.get_type_name() == "Convolution" for op in model.get_ops()):
+        wc_config.ignored_scope["types"] = ["Convolution"]
+
+    q_config_ignored_scope = {"names": _collect_ops_with_weights(model)}
+    q_config = OVQuantizationConfig(
+        ignored_scope=q_config_ignored_scope,
+        num_samples=quantization_config.num_samples or 200,
+        smooth_quant_alpha=-1,
```

Comment:
```suggestion
        # SQ algo should be disabled for MatMul nodes because their weights are already compressed
        advanced_parameters=nncf.AdvancedQuantizationParameters(
            smooth_quant_alphas=AdvancedSmoothQuantParameters(matmul=-1)
        ),
```

Reply:
What about other quantization parameters, such as `model_type=nncf.ModelType.TRANSFORMER`?

Reply:
There is possibly some misunderstanding here. `q_config` is an instance `OVQuantizationConfig` which accepts SQ alpha as `smooth_quant_alpha` argument, because this is our wrapper over nncf quantization parameters. Also, model_type is set to "transformer" by default there.